ajax_with_jquery_exercise/hack_clone.js

```diff
@@ -0,0 +1,37 @@
+$(function() {
```

Comment:
Good start.  Keep going on this when you can.  It's a good jquery/ajax exercise.